src/SuperSocket.Connection/PipeConnectionBase.cs

```diff
@@ -257,6 +281,24 @@ private void WriteBuffer(PipeWriter writer, ReadOnlyMemory<byte> buffer)
             writer.Write(buffer.Span);
         }
 
+        private void WriteBuffer(PipeWriter writer, ReadOnlySequence<byte> buffer)
+        {
+            CheckConnectionSendAllowed();
+            //writer.Write(buffer.Span);
+            if (buffer.IsSingleSegment)
+            {
+                writer.Write(buffer.First.Span);
+            }
+            else
+            {
+                foreach (var memory in buffer)
+                {
+                    writer.Write(memory.Span);

```

Comment:
Better have some flushes in the middle. If we only flush in the end when we send a large sequence, it would cause memory leak and sending delay.